### PR TITLE
Adding Changelog URL to the README.md file

### PR DESCRIPTION
**Pull request by dahabit**
Adding Changelog URL to the README.md file

### CODE REVIEW

**Comment by Hixie** [comment]
This is a great idea!

Looks like the other two lines shouldn't have been changed, though. Any chance you can change them back to what they were before?

**Comment by dahabit** [comment]
> This is a great idea!
> 
> Looks like the other two lines shouldn't have been changed, though. Any chance you can change them back to what they were before?

Thanks, @Hixie  for your reply, I added a new line for the changelog link, and it pushed the other lines one line down I didn't change anything just added 1 line

**Comment by Hixie** [comment]
According to the diff on GitHub, line ten changed from `* [Install](https://flutter.io/get-started/install/)` to `* [Install](https://flutter.io/setup/)`. Maybe this was something that changed while your branch was open or something? Anyway if you could change those to match what's on the trunk now that would let us land this PR. Thanks again for your contribution!

**Comment by dahabit** [comment]
Hello @Hixie I pulled the latest version on dev branch and added one line for the changelog URL, please let me know if there are any other changes needed

File: README.md [hunk 1]
```diff
@@ -12,7 +12,8 @@ Flutter is Google’s mobile app SDK for crafting high-quality native interfaces
 * **Main site: [flutter.io][]**
 * [Install](https://flutter.io/get-started/install/)
 * [Get started](https://flutter.io/get-started/)
-* [Contribute](https://github.com/flutter/flutter/blob/master/CONTRIBUTING.md#contributing-to-flutter)
+* [Changelog](https://github.com/flutter/flutter/wiki/Changelog)
+* [Contribute](CONTRIBUTING.md)
```
Comment (dnfield) [inline README.md]: This line needs to be fixed.

**Comment by dnfield** [comment]
@dahabit I've gone ahead and fixed a broken link and merged this with upstream - LGTM now!  Thanks for the contribution!

**Comment by dahabit** [comment]
@dnfield  thanks for your effort, and sorry for the last response , yeah I think CI is messed up :D

**Pull request merged**